pkg/kubelet/kubelet_node_status.go

```diff
@@ -172,6 +175,36 @@ func (kl *Kubelet) reconcileHugePageResource(initialNode, existingNode *v1.Node)
 	return requiresUpdate
 }
 
+// reconcileSwapResource will update swap capacity and remove swap size no longer supported
```

Comment:
I don't quite understand this comment, I think this was a copy paste from above.

```suggestion
// reconcileSwapResource will update swap capacity and remove swap if the feature is disabled
```